WordPress/Resources/release_notes.txt

```diff
@@ -1,17 +1,11 @@
-What’s new in WordPress release 17.5?
+* [**] Reader Post details: now shows a summary of Likes for the post. Tapping it displays the full list of Likes. [#16628]
```

Comment:
Ideally I think we'd get rid of the priority indicators and PR numbers in this file prior to landing it, but I don't recall if this is a tooling-generated change.

No need to fix it for this PR, but if it was by hand, it'd be nice for next time – just makes the release notes a bit nicer in GitHub and such :)

Reply:
Gotcha! I'll keep it in mind 👌